src/utils/index.ts

```diff
@@ -1743,7 +1745,7 @@ export function getTokenFromAddress(
     return token;
   }
 
-  return wrappedTokenInfo;
+  return Object.values(tokenMap[chainId])[tokenIndex];
```

Comment:
We're sure this isn't used as a dependency in any hooks cuz it will cause an infinite loops if so

Reply:
This is not used in hook so it will not cause infinite loop.